cmd/src/snapshot_databases.go

```diff
@@ -94,19 +96,31 @@ TARGETS FILES
 				out.WriteLine(output.Emojif(output.EmojiInfo, "Using predefined targets for %s environments", targetKey))
 			}
 
-			commands, err := pgdump.BuildCommands(srcSnapshotDir, commandBuilder, targets)
+			commands, err := pgdump.BuildCommands(srcSnapshotDir, commandBuilder, targets, true)
 			if err != nil {
 				return errors.Wrap(err, "failed to build commands")
 			}
 
 			_ = os.MkdirAll(srcSnapshotDir, os.ModePerm)
 
-			b := out.Block(output.Emoji(output.EmojiSuccess, "Run these commands to generate the required database dumps:"))
-			b.Write("\n" + strings.Join(commands, "\n"))
-			b.Close()
+			if *run {
+				for _, c := range commands {
+					out.WriteLine(output.Emojif(output.EmojiInfo, "Running command: %q", c))
+					command := exec.Command("bash", "-c", c)
+					out, err := command.CombinedOutput()
```

Comment:
I forget, how much output does pgdump and friends generate? If it gives some progress info we might want to send output to stdout/stderr instead of collecting them with `CombinedOutput`

Reply:
Restore was pretty noisy so now restore stdout is redirected to /dev/null
Only stderr is captured